storage/tools/dump_tree/dumplib/dumplib.go

```diff
@@ -256,23 +255,22 @@ func Main(args Options) string {
 	sequenceLeaves(ls, seq, tree.TreeId, args.TreeSize, args.BatchSize, args.LeafFormat)
 
 	// Read the latest STH back
-	tx, err := ls.BeginForTree(context.TODO(), tree.TreeId)
-	if err != nil {
-		glog.Fatalf("BeginForTree got: %v, want: no err", err)
-	}
+	var sth trillian.SignedLogRoot
+	err := ls.ReadWriteTransaction(context.TODO(), tree.TreeId, func(ctx context.Context, tx storage.LogTreeTX) error {
```

Comment:
SnapshotForTree? Doesn't really matter as this is test code really.

Reply:
Yeah, I just converted it as-was.